internal/security/credentials_test.go

```diff
@@ -41,13 +43,20 @@ type getSecretsTestObj struct {
 }
 
 var getSecretsTestData = []getSecretsTestObj{
-
+	getSecretsTestObj{
```

Comment:
Add one more where the path = "". I am assuming this will be the common case for custom app services. I assume path = "" is valid in vault??